### ISSUE TITLE
Add "Ping Server" Button to UI

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#292]
```
Since the PingManager is now in place, we could add a "Ping Server" button to 
the UI, which triggers a ping and shows a indicator of the ping status: 
success (green)
waiting for reply (orange)
no response (red)

I the latter case the pingFailedListeners will automatically be called, so that 
we can take action upon it
```

Original issue reported on code.google.com by `fschm...@gmail.com` on 28 Aug 2012 at 12:39

**Comment by GoogleCodeExporter** [comment]
```
Will be released with GTalkSMS 4.3
```

Original comment by `fschm...@gmail.com` on 11 Nov 2012 at 11:21
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 4.3 release
```

Original comment by `fschm...@gmail.com` on 26 Jan 2013 at 5:45
- Changed state: **Deployed**
